types/sdk.js

```diff
@@ -60,7 +60,7 @@ export type AuthMiddlewareOptions = {
   scopes: Array<string>;
 }
 export type HttpMiddlewareOptions = {
-  host?: string;
+  host: string;
```

Comment:
Well as far as I could see not since we're always assigning a value to it, otherwise Flow will complain.